Homework-3/Homework-3/ViewController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  ViewController.swift
+//  Homework-3
+//
+//  Created by Kirill Skrytskii on 5.03.22.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        
+        sum(2.3, to: 1.6)
+        sum(6, to: 2, to: 3)
+        sum(30.2, by: 2)
+        let totalSumTupes = sumTupes(23.2, to: 43)
+        print(totalSumTupes)
+        
+        let sumDigits = digitSum(to: 2022)
+        print(sumDigits)
+        
+        string("Hello", to: "Hello")
+        print(power())
+        
+        let factorial = recursionFactorial(6)
+        print(factorial)
+    }
+    
+    // 1
+    
+    func sum(_ firstNumber: Double, to secondNumber: Double) {
```

Comment:
Good job with parameters naming here. I can propose an improvement here. When we say in English about sum, we say a sum of a first number and a second number but we say add a first number to a second number. So we can have `sum(of firstNumber: Double, and secondNumber: Double))` or `add(_ firstNumber: Double, to secondNumber: Double)` here